specification/sections/02-providers.md

```diff
@@ -85,10 +85,12 @@ As indicated in the definition of the [`flag resolution`](../types.md#resolution
 
 ##### Requirement 2.8
 
-> In cases of abnormal execution, the `provider` **MUST** indicate an error using the idioms of the implementation language, with an associated error code having possible values `"PROVIDER_NOT_READY"`, `"FLAG_NOT_FOUND"`, `"PARSE_ERROR"`, `"TYPE_MISMATCH"`, or `"GENERAL"`.
+> In cases of abnormal execution, the `provider` **MUST** indicate an error using the idioms of the implementation language, with an associated `error code`.
```

Comment:
I feel like there should be something about the `error message` here as well. Or a similar requirement that includes the message. I am not sure how that would be forumated.